nixos/tests/rspamd.nix

```diff
@@ -320,7 +320,6 @@ in
 
       machine.wait_until_fails('[ "$(postqueue -p)" != "Mail queue is empty" ]')
       machine.fail("journalctl -u postfix | grep -i error >&2")
-      machine.fail("journalctl -u postfix | grep -i warning >&2")
```

Comment:
What about filtering out the new warning message instead of ignoring all of them?

(If you remove it because it's annoying to maintain, i'm ok to with this reason.)

Reply:
The warnings are due to the permissions of the config files. I haven't seen a problem with rspamd yet that only triggers a warning, so I think it should be fine to ignore warnings for now.